### PR TITLE
Remove @<hostname> from ReqMgr2 Requestor field

### LINKED ISSUE #6637: ReqMgr2 request name structure
Antanas created a new reqmgr2 request today:
pdmvserv@vocms081.cern.ch_SUS-RunIIWinter15pLHE-00232_00146_v1__160226_103421_890

using this DN:
https://cmsweb.cern.ch/sitedb/prod/people/pdmvserv%40vocms081.cern.ch

Since we don't have the same username registration as we do for reqmg (oracle), I believe we are somehow getting the sitedb "username" or so, to prepend to the request name.

### PR DESCRIPTION
**Pull request by amaltaro**
Fixes #6637 

Still needs to be tested

### CODE REVIEW

File: src/python/WMCore/ReqMgr/DataStructs/Request.py [hunk 1]
```diff
@@ -21,66 +21,67 @@
...
-def initialize_request_args(request, config, clone = False):
+
+def initialize_request_args(request, config, clone=False):
     """
     Request data class request is a dictionary representing
     a being injected / created request. This method initializes
     various request fields. This should be the ONLY method to
     manipulate request arguments upon injection so that various
-    levels or arguments manipulation does not occur accros several
+    levels or arguments manipulation does not occur accross several
```
Comment (ericvaandering) [inline src/python/WMCore/ReqMgr/DataStructs/Request.py]: Close. Across.

Comment (amaltaro) [inline src/python/WMCore/ReqMgr/DataStructs/Request.py]: Darn it! :)

File: src/python/WMCore/ReqMgr/DataStructs/Request.py [hunk 2]
```diff
@@ -21,66 +21,67 @@
...
-    
-    #user information for cert. (which is converted to cherry py log in)
+    """
+
+    # user information for cert. (which is converted to cherry py log in)
     request["Requestor"] = cherrypy.request.user["login"]
     request["RequestorDN"] = cherrypy.request.user.get("dn", "unknown")
-    
+    # service certificates carry @hostname, let us remove it
+    request["Requestor"] = request["Requestor"].split('@')[0]
```
Comment (ericvaandering) [inline src/python/WMCore/ReqMgr/DataStructs/Request.py]: Remove it if it exists. Which this line does.

**Comment by amaltaro** [comment]
Antanas (thanks!) helped me to validate it. Merging

**Pull request merged**